### PR TITLE
MGMT-1736 Add CreatedAt column to clusters list

### PR DESCRIPTION
**Pull request by jtomasek**
~~Depends on https://github.com/mareklibra/facet-lib/pull/63~~

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -100,6 +99,8 @@
     "library"
   ],
   "dependencies": {
+    "@types/human-date": "^1.4.0",
```
Comment (rawagner) [inline package.json:102]: this should be `devDependency`

File: src/components/clusters/ClustersTable.tsx [hunk 2]
```diff
@@ -40,6 +40,8 @@ const columns = [
   { title: 'Version', ...columnConfig },
   { title: 'Status', ...columnConfig },
   { title: 'Hosts', ...columnConfig },
+  { title: 'Status Updated At', ...columnConfig },
+  { title: 'Created At', ...columnConfig },
```
Comment (andybraren) [inline src/components/clusters/ClustersTable.tsx:44]: If the date format we use is "2 days ago" and eventually a date, `Created` without the `at` may be better here, which is what OCM uses. Their `Created` is a bit different than ours, but I'm struggling to find out a better word.

Maybe our `database_created_date` could be shown as the `Created` date while the cluster is a Draft, and after it succeeds the `installation_finished_time` could be used as the `Created` date instead? WDYT?

File: package.json (hunk 1 above)
Comment (jtomasek) [reply package.json]: done

File: src/selectors/clusters.tsx [hunk 3]
```diff
@@ -48,6 +48,7 @@ const clusterToClusterTableRow = (cluster: Cluster): IRow => {
         title: hostsCount.toString(),
         sortableValue: hostsCount,
       } as HumanizedSortable,
+      getRelativeTimeCell(createdAt),
```
Comment (mareklibra) [inline src/selectors/clusters.tsx:51]: I think the user should be able to see even non-relative time. In a popover? @andybraren 

So far all attempts to show relative time throughout the application were reverted back to rendering real time. How is this different?

**Comment by jtomasek** [comment]
Reverted back to not using relative time
Screenshot 2020-09-30 at 10 26 39

**Review by mareklibra — APPROVED** [review]

**Review by mareklibra — APPROVED** [review]

**Pull request merged**